chat/templates/chat/chat.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'main/loggedInPage.html' %}
+{% block main_content %}
+{% load static %}
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
+<script>
+    var staticPhotosFolder = "{% static "images/" %}";
+    var currentUserID = {{request.user.id}};
+    var csrf_token = '{{ csrf_token }}';
+</script>
```

Comment:
Maybe it can be moved to one of the JS files

Reply:
As I found this is the option that is available in order to use django variables in a separated JS file (the assignment must be in the template for it to render)